third_party/CMakeLists.txt

```diff
@@ -11,5 +11,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-add_subdirectory(googletest)
+
+if(VELOX_ENABLE_GOOGLETEST)
```

Comment:
Why not use VELOX_BUILD_TESTING here?

Reply:
Since googletest is needed by both `VELOX_BUILD_TESTING` and `VELOX_ENABLE_BENCHMARKS_BASIC` since some of the benchmarks are coupled with the test classes.
So `VELOX_ENABLE_GOOGLETEST` is a proxy for either.